io-uring-test/src/tests/register_buf_ring.rs

```diff
@@ -664,6 +664,15 @@ fn buf_ring_play<S: squeue::EntryMarker, C: cqueue::EntryMarker>(
     normal_check(&buf3, 1); // bid 1 should come back first.
     normal_check(&buf4, 0); // bid 0 should come back second.
 
+    std::mem::drop(buf3);
+    std::mem::drop(buf4);
+
+    // Now we loop u16::MAX+1 times to ensure proper behavior when the tail
+    // overflows the bounds of a u16.
+    for _ in 0..(u16::MAX as usize + 1) {
```

Comment:
You can use `0..=u16::MAX`.

Reply:
Ah! Good point. I've fixed this in [a7aba2a](https://github.com/tokio-rs/io-uring/pull/243/commits/a7aba2a735a8bd72d5c85cab5060f1a8653d1294).